goji/commands.py

```diff
@@ -308,13 +308,25 @@ def search(client, format, query):
     issues = client.search(query)
 
     for issue in issues:
-        click.echo(format.replace('\\n', '\n').format(
+        # LKBM
+        status = str(issue.status)
+        if status in ['Open', 'To Do', 'Subscriber Ux to do', 'Merchant operations to do']:
```

Comment:
This seems a bit specific to your particular workflow, I wonder if we can use statusCategory instead of the specific status instead. What do you think?

Reply:
Yeah, oops. I meant this to be a pr against my own fork. It is very specific to my Jira setup.